buildout.cfg

```diff
@@ -1,3 +1,3 @@
 [buildout]
 extends = base.cfg
-develop = ..
```

Comment:
@rnix btw.: this one here took me 30min to figure out... Why? Lets stick to the standards.